pkg/iostreams/color.go

```diff
@@ -9,41 +9,53 @@ import (
 )
 
 const (
+	NoTheme        = "none"
+	DarkTheme      = "dark"
+	LightTheme     = "light"
 	highlightStyle = "black:yellow"
 )
 
+// Special cases like darkTableHeader / lightTableHeader are necessary when using color and modifiers
+// (bold, underline, dim) because ansi.ColorFunc requires a foreground color and resets formats.
 var (
-	magenta            = ansi.ColorFunc("magenta")
-	cyan               = ansi.ColorFunc("cyan")
-	red                = ansi.ColorFunc("red")
-	yellow             = ansi.ColorFunc("yellow")
-	blue               = ansi.ColorFunc("blue")
-	green              = ansi.ColorFunc("green")
-	gray               = ansi.ColorFunc("black+h")
-	lightGrayUnderline = ansi.ColorFunc("white+du")
-	bold               = ansi.ColorFunc("default+b")
-	cyanBold           = ansi.ColorFunc("cyan+b")
-	greenBold          = ansi.ColorFunc("green+b")
-	highlightStart     = ansi.ColorCode(highlightStyle)
-	highlight          = ansi.ColorFunc(highlightStyle)
+	magenta          = ansi.ColorFunc("magenta")
+	cyan             = ansi.ColorFunc("cyan")
+	red              = ansi.ColorFunc("red")
+	yellow           = ansi.ColorFunc("yellow")
+	blue             = ansi.ColorFunc("blue")
+	green            = ansi.ColorFunc("green")
+	gray             = ansi.ColorFunc("black+h")
+	bold             = ansi.ColorFunc("default+b")
+	cyanBold         = ansi.ColorFunc("cyan+b")
+	greenBold        = ansi.ColorFunc("green+b")
+	highlightStart   = ansi.ColorCode(highlightStyle)
+	highlight        = ansi.ColorFunc(highlightStyle)
+	darkTableHeader  = ansi.ColorFunc("white+du")
+	lightTableHeader = ansi.ColorFunc("black+hu")
+	noneTableHeader  = ansi.ColorFunc("default+u")
```

Comment:
Suggestion: Would you open to some small inline comments here to help indicate that `dark` is referring to the theme, not a color as the other color vars do? 

```suggestion
	darkTableHeader  = ansi.ColorFunc("white+du") // for dark theme
	lightTableHeader = ansi.ColorFunc("black+hu") // for light theme
	noneTableHeader  = ansi.ColorFunc("default+u")
```

Reply:
Solid suggestion; rather than adding the inline comment, I went with renaming the variables to communicate their theme relationship explicitly.